README.md

```diff
@@ -112,20 +91,21 @@ Current release info
 | Name | Downloads | Version | Platforms |
 | --- | --- | --- | --- |
 | [![Conda Recipe](https://img.shields.io/badge/recipe-scikit--image-green.svg)](https://anaconda.org/conda-forge/scikit-image) | [![Conda Downloads](https://img.shields.io/conda/dn/conda-forge/scikit-image.svg)](https://anaconda.org/conda-forge/scikit-image) | [![Conda Version](https://img.shields.io/conda/vn/conda-forge/scikit-image.svg)](https://anaconda.org/conda-forge/scikit-image) | [![Conda Platforms](https://img.shields.io/conda/pn/conda-forge/scikit-image.svg)](https://anaconda.org/conda-forge/scikit-image) |
+| [![Conda Recipe](https://img.shields.io/badge/recipe-scikit--image--core-green.svg)](https://anaconda.org/conda-forge/scikit-image-core) | [![Conda Downloads](https://img.shields.io/conda/dn/conda-forge/scikit-image-core.svg)](https://anaconda.org/conda-forge/scikit-image-core) | [![Conda Version](https://img.shields.io/conda/vn/conda-forge/scikit-image-core.svg)](https://anaconda.org/conda-forge/scikit-image-core) | [![Conda Platforms](https://img.shields.io/conda/pn/conda-forge/scikit-image-core.svg)](https://anaconda.org/conda-forge/scikit-image-core) |
 
-Installing scikit-image
-=======================
+Installing scikit-image-core
+============================
 
-Installing `scikit-image` from the `conda-forge` channel can be achieved by adding `conda-forge` to your channels with:
+Installing `scikit-image-core` from the `conda-forge` channel can be achieved by adding `conda-forge` to your channels with:
 
 ```
 conda config --add channels conda-forge
 ```
 
-Once the `conda-forge` channel has been enabled, `scikit-image` can be installed with:
+Once the `conda-forge` channel has been enabled, `scikit-image, scikit-image-core` can be installed with:
```

Comment:
@hmaarrfk so you're saying that this is auto-generated? That's probably not desirable in this case... =\

Reply:
Yeah, this is auto generated.

I would consider conda-forge more bleeding edge. Anaconda Defaults regularly syncs up with conda-forge so the work we do here eventually gets upstreamed to there..

Reply:
I also do not think that anybody is going on this page to learn how to use "scikit-image", just the packages. scikit-image's docs are still valid, and shouldn't mention core. Packages should be able to read this "meta.yaml" file and understand what is going on.